variables.tf

```diff
@@ -66,6 +66,18 @@ variable "resource" {
   description = "Hive resources"
 }
 
+variable "resource_proxy" {
+  type = object({
+    cpu     = number,
+    memory  = number
+  })
+  default = {
+    cpu     = 300,
+    memory  = 512
+  }
+  description = "Hive proxy resources"
+}
+
```

Comment:
If these values are overwritten with smaller values, will the proxy fail? In that case, we should add a `validation` block. Something like this:
```hcl
validation {
    condition     = var.resource_proxy.cpu >= 300 && var.resource_proxy.memory >= 512
    error_message = "Proxy resource must be at least: cpu=300, memory=512"
}
```

Reply:
Ah, absolutely, yes! Thank you, I completely forgot about this 😬

Reply:
I’ve tested this (in vagrant-box), and found that the metastore-proxy will work with such small values; `cpu = 20` and `memory = 80`. But I think we should set `validation`-stanza to default values (`cpu = 200` and `memory = 128`). I will fix.

Reply:
I've [added validation-stanza](https://github.com/fredrikhgrelland/terraform-nomad-hive/pull/55/commits/29e619007e1c08e2382ab8ddc8ee3b196d3098d7) to hive.hcl.